doc/source/templates/04_finetuning_llms_with_deepspeed/create_job_yaml.py

```diff
@@ -0,0 +1,79 @@
+from argparse import ArgumentParser
+
+import yaml
+import os
+import pathlib
+
+
+def _parse_args():
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--size",
+        type=str,
+        default="7b",
+        choices=["7b", "13b", "70b"],
+        help="Size of the model to train",
+    )
+    parser.add_argument(
+        "--as-test", action="store_true", help="Whether to run in test mode"
+    )
+    parser.add_argument(
+        "--max-retries",
+        type=int,
+        default=0,
+        help="Number of times to retry the job if it fails",
+    )
+    parser.add_argument(
+        "--output-path",
+        type=str,
+        default="./job.yaml",
+        help="The path that job yaml should be stored.",
+    )
+    parser.add_argument("--compute-config", type=str, help="Path to the compute config")
+    parser.add_argument(
+        "--cluster-env-build-id",
+        type=str,
+        help="The build-id of the cluster env to use",
+    )
```

Comment:
Are there default values for these? (The README update indicates that there should be)

Reply:
default is None for this one.